public/locales/en-US/translations.json

```diff
@@ -369,14 +367,15 @@
   },
   "badConnectionModal": {
     "title": "Connection issue",
-    "text1": "We've noticed several internet connection issues. It's required to {{action}} the app to continue normal operation.",
-    "text2": "Please make sure you have stable and good internet connection.",
-    "text3": "The Bitfinex Honey will {{action}} after you press 'Okay'.",
-    "text4": "The app will {{action}} automatically in {{countdown}} seconds.",
+    "description": "There has been a connection issue, all running strategies and algorithmic orders have been cancelled.",
     "checkbox": "{{action}} automatically",
-    "reboot": "reboot",
+    "reboot": "Reboot",
     "restart": "Restart"
   },
+  "noConnectionActionModal": {
+    "title": "Connection issue",
+    "description": "This action can't be run because we the app is encountering connection issues.\nWe are trying to reconnect automatically.\nTry again once you see 'WS connected' present on the status bar at the bottom right-corner."
```

Comment:
```suggestion
    "description": "This action can't be run because the app is encountering connection issues.\nWe are trying to reconnect automatically.\nTry again once you see 'WS connected' present on the status bar at the bottom right-corner."
```